pkg/operator/ceph/cluster/osd/labels.go

```diff
@@ -36,13 +36,19 @@ const (
 	OSDOverPVCLabelKey = "ceph.rook.io/pvc"
 	// TopologyLocationLabel is the crush location label added to OSD deployments
 	TopologyLocationLabel = "topology-location-%s"
+	// CephImageVersionLabel is the ceph image version label added to PVC
+	CephImageLabelKey = "ceph.rook.io/CreatedCephImage"
```

Comment:
> That's true. The label stores image name not just a image tag. And I prefer `Created` rather than `Original` because I intended the label to indicate the image at the point of the creation of OSD, so I will change it to `CreatedCephImage` and `CreatedRookImage`

Agreed, `created` is better, and I think "AtCreation" is most clear. How about `cephImageAtCreation` and `rookImageAtCreation`? Also I believe all our labels currently start with lower case.

Reply:
>The repo likely won't change over time, right? But that's ok to keep the full image name if you prefer.

For example, there is a possibility of moving from quay.io to ghcr.io. So I want to store the full image name.

>Agreed, created is better, and I think "AtCreation" is most clear. How about cephImageAtCreation and rookImageAtCreation? Also I believe all our labels currently start with lower case.

It's sounds nice. I'll change codes to use those.